pkg/console/subresource/deployment/deployment.go

```diff
@@ -238,15 +238,16 @@ func consoleVolumeMounts(vc []volumeConfig) []corev1.VolumeMount {
 
 func GetLogLevelFlag(logLevel operatorv1.LogLevel) string {
 	flag := ""
-	// Since the console-operator logging has different logging levels then the capnslog,
-	// that we use for console server(bridge) we need to map them to each other
+	// console server(bridge) expects flag log-level
 	switch logLevel {
 	case operatorv1.Normal:
-		flag = "--log-level=*=NOTICE"
+		flag = "--log-level=2"
```

Comment:
These are still 

```
--log-level=2
```
But we need to update to 

```
-v=2
```

Wrong flag.

Reply:
I am leaving this like this for now since console can handle the old values.  We will get console portion merged first and then update this.  I will update the code but I want to make sure this does not get merged.

Reply:
This PR won't merge until the `-v` flag is updated, so you should be safe.